web-app/src/components/LoginForm/container.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import Swal from 'sweetalert2';
+import { PropTypes as PT } from 'prop-types';
+import { userLogin } from '../../services/userService';
+
+export default Wrapped =>
```

Comment:
Can we name this something that better explains what it is? `Wrapped` is a bit abstract

Reply:
Wrapped is what has been used everywhere in the project that has this pattern. I was just following the convention.

Reply:
fair enough, not a big fan of it but it's cool

Reply:
I agree. I'll put it to the group and see if we can make it a fix throughout the project on another pull request.